### PR TITLE
Adding reference of nodejs adapter usage from keycloak quickstart example

### PR DESCRIPTION
**Pull request by pacostas**

### CODE REVIEW

File: securing_apps/topics/oidc/nodejs-adapter.adoc [hunk 1]
```diff
@@ -313,3 +313,7 @@ to the root URL of `/` but can be changed by providing an `admin` parameter
 to the `middleware()` call:
 [source,javascript]
     app.use( keycloak.middleware( { admin: '/callbacks' } );
+
+==== Additional examples 
+
+Another example of Node.js adapter usage can be found on https://github.com/keycloak/keycloak-quickstarts/tree/latest/service-nodejs/[keycloack quickstarts for nodejs]
```
Comment (mhdawson) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
==== Complete example 

A complete example using the Node.js adapter usage can be found in https://github.com/keycloak/keycloak-quickstarts/tree/latest/service-nodejs/[keycloack quickstarts for nodejs]
```

Comment (mhdawson) [reply securing_apps/topics/oidc/nodejs-adapter.adoc]: @pacostas just a few suggestions in terms of wording but this looks like a good place to include a reference.

File: securing_apps/topics/oidc/nodejs-adapter.adoc [hunk 2]
```diff
@@ -313,3 +313,7 @@ to the root URL of `/` but can be changed by providing an `admin` parameter
 to the `middleware()` call:
 [source,javascript]
     app.use( keycloak.middleware( { admin: '/callbacks' } );
+
+==== Complete example 
+
+A complete example using the Node.js adapter usage can be found in https://github.com/keycloak/keycloak-quickstarts/tree/latest/service-nodejs/[keycloack quickstarts for nodejs]
```
Comment (abstractj) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]: We have a typo here. keycloack vs Keycloak.

Please replace by Keycloak quickstarts for Node.js

**Review by abstractj — CHANGES_REQUESTED** [review]
Please see my comments.

File: securing_apps/topics/oidc/nodejs-adapter.adoc [hunk 3]
```diff
@@ -313,3 +313,7 @@ to the root URL of `/` but can be changed by providing an `admin` parameter
 to the `middleware()` call:
 [source,javascript]
     app.use( keycloak.middleware( { admin: '/callbacks' } );
+
+==== Complete example 
+
+A complete example using the Node.js adapter usage can be found in https://github.com/keycloak/keycloak-quickstarts/tree/latest/service-nodejs/[keycloak quickstarts for nodejs]
```
Comment (abstractj) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]: Please replace ```nodejs``` by Node.js

**Review by abstractj — DISMISSED** [review]
Changes look good to me and before merging it I can squash all the commits in a single one.

File: securing_apps/topics/oidc/nodejs-adapter.adoc [hunk 4]
```diff
@@ -313,3 +313,7 @@ to the root URL of `/` but can be changed by providing an `admin` parameter
 to the `middleware()` call:
 [source,javascript]
     app.use( keycloak.middleware( { admin: '/callbacks' } );
+
+==== Complete example 
+
+A complete example using the Node.js adapter usage can be found in https://github.com/keycloak/keycloak-quickstarts/tree/latest/service-nodejs/[keycloak quickstarts for Node.js]
```
Comment (mhdawson) [inline securing_apps/topics/oidc/nodejs-adapter.adoc]:
```suggestion
A complete example using the Node.js adapter usage can be found in https://github.com/keycloak/keycloak-quickstarts/tree/latest/service-nodejs/[Keycloak quickstarts for Node.js]
```

**Comment by mhdawson** [comment]
@abstractj I think all your updated requests where addressed.  I think the external links failures are not related to this PR.

**Review by abstractj — APPROVED** [review]

**Comment by abstractj** [comment]
@mhdawson thank you for all the submissions!

**Pull request merged**